### DISCUSSION TITLE
Display images in alphabetical order by file name.

### DISCUSSION THREAD

**Discussion by ai30328348** [immich-app/immich#14683] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Display images in alphabetical order by file name.
I feel that sorting only by the creation date of the images limits many usage methods. The stacking function requires manual merging of the images, which is not very convenient. I have already organized the file names in the format of image name + sequence number, but these two images are not nearby. It is not feasible for me to modify the creation time of each image, and changing the file names is easier than modifying the creation time, so I hope to add this functionality.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4170
